trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/index.ts

```diff
@@ -25,7 +27,12 @@ export const addToAnExistingPayment = async (req: AddToAnExistingPaymentRequest,
     const checkedCheckboxIds = getPremiumPaymentsCheckboxIdsFromObjectKeys(req.body);
     const feeRecordIds = getFeeRecordIdsFromPremiumPaymentsCheckboxIds(checkedCheckboxIds);
 
-    const selectedFeeRecordDetails = await api.getSelectedFeeRecordsDetails(reportId, feeRecordIds, userToken);
+    const selectedFeeRecordDetails = await api.getSelectedFeeRecordsDetailsWithAvailablePaymentGroups(reportId, feeRecordIds, userToken);
```

Comment:
Could we throw an error here if `canAddToExistingPayment` is false? The reason being it should be pretty easy to add in and guards against someone trying to access the page for fee records not in say the TO DO state for example, failing the journey as early as possible

Reply:
Omitting this now that we have the check below on the value of availablePaymentGroups:

`if (!availablePaymentGroups || availablePaymentGroups.length === 0) {`